Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -503,6 +513,34 @@ def audit_deps
           problem "Formulae in homebrew/core should not have optional or recommended requirements"
         end
       end
+
+      return unless @core_tap
+      return if VERSIONED_DEPENDENCIES_CONFLICTS_ALLOWLIST.include?(formula.name)
+
+      # The number of conflicts on Linux is absurd.
+      # TODO: remove this and check these there too.
+      return if OS.linux?
```

Comment:
CC @Homebrew/linux that I did this. Uncomment it and try it out to see for yourself 🙈